spec/gem_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+RSpec.describe Rootstrap::Gem do
+  gem_name = 'test'
+
+  describe '#create' do
+    before(:all) { described_class.new(gem_name: gem_name).create }
```

Comment:
I'm not too fond on having the *Act* section of the test inside a *before*... Are you doing it to reduce the time the suite takes to run?

Reply:
Well I can update it to be the subject, but given this is actually calling `bundle` in the system and also copy/pasting contents of configurations files I figured this way it'd be faster to test all behaviours involved. This can be seen as an integration test given that this is the entry point or main class in the gem.

What do you think if keep doing this in this before hook for now?

Reply:
Yeah, I sure don't mind! In this case it would do more harm to repeat the action for every test rather than leaving it in the before.